engine/servermaster/server.go

```diff
@@ -21,28 +21,20 @@ import (
 	"net/http"
 	"time"
 
+	serverutil2 "github.com/pingcap/tiflow/engine/servermaster/serverutil"
```

Comment:
Why put this import on a separate line? BTW, do we have a style guide for import group, three groups or two groups.